README.md

```diff
@@ -1,28 +1,89 @@
-# Simularium Conversion
+# SimulariumIO
 
 [![Build Status](https://github.com/allen-cell-animated/simulariumio/workflows/Build%20Master/badge.svg)](https://github.com/allen-cell-animated/simulariumio/actions)
 [![Documentation](https://github.com/allen-cell-animated/simulariumio/workflows/Documentation/badge.svg)](https://allen-cell-animated.github.io/simulariumio)
 [![Code Coverage](https://codecov.io/gh/allen-cell-animated/simulariumio/branch/master/graph/badge.svg)](https://codecov.io/gh/allen-cell-animated/simulariumio)
 
-Simularium Conversion helps convert ssimulation outputs to the format consumed by the Simularium viewer.
+Simulariumio converts simulation outputs to the format consumed by the [Simularium viewer](https://simularium.allencell.org/).
 
 ---
 
 ## Features
-* Store values and retain the prior value in memory
-* ... some other functionality
+* Converts 3D spatiotemporal trajectories to .simularium JSON format
+* Accepts spatial trajectories from the following biological simulation engines:
+    * CytoSim (https://gitlab.com/f.nedelec/cytosim)
+    * ReaDDy (https://readdy.github.io/)
+    * PhysiCell (http://physicell.org/)
+    * Custom engines not specifically supported
```

Comment:
Maybe say something like "Conversions for custom engine data can be implemented using a base converter class."  if that is accurate?